boruta/readme.md

```diff
@@ -0,0 +1,30 @@
+# Boruta Feature Selection
+
+In this package you'll find two scripts implementing Boruta Feature Selection:
+
+- [Boruta 1-click Feature Selection](./1-click-boruta)
+  Starting from a dataset, creates a new one removing the unimportant features
+  according to the Boruta Feature Selection algorithm.
+- [Boruta Feature Selection](./boruta)  General Boruta Feature Selection
+  algorithm to rank fields from a dataset into three categories: important,
+  unimportant and undecided.
```

Comment:
what's the difference with the previous one? i would have the 1-click-boruta, and add a library when we can...